src/Mollie/WC/Plugin.php

```diff
@@ -1103,5 +1103,21 @@ private static function extractRemoteItemsIds(array $data)
 
         return implode(',', wp_list_pluck($data['lines'], 'id'));
     }
+
+    /**
+     * @return array
+     */
+    public static function paymentMethodsImages()
```

Comment:
From the internal of this method I don't not see Images context.

Are you returning whatever you get from the Api? If so what that response contains?